core/server/data/meta/asset_url.js

```diff
@@ -39,11 +39,13 @@ function getAssetUrl(path, isAdmin, minify) {
         // we don't want to concat the path with our favicon url
         output += path;
 
-        if (!config.get('assetHash')) {
-            config.set('assetHash', utils.generateAssetHash());
-        }
+        if (config.get('env') === 'production') {
```

Comment:
At any rate - I think I'll remove this from the PR -> it _shouldn't_ cause issues with caching and we can re-evaluate disabling it in dev mode at a later time.